config/locales/en/providers.yml

```diff
@@ -214,3 +214,18 @@ en:
     success_prospects:
       show:
         h1-heading: What are the prospects of success?
+    capital_introductions:
+      show:
+        list: |
+          owns a home or has a mortgage (and their values)
+          has any savings, investments or other capital (and their values)
+          has any other ways to contribute towards legal costs
+        list_2:
+          list: |
+            a statement of case
+            information about any attempts to settle
+            cost estimates
+            the prospects of success
+        h1-heading: Before you continue
+        text_1: "You'll need to tell us if your client:"
+        text_2: "You'll also need to provide:"
```

Comment:
`list_2`, `text_1` and `text_2` could be made meaningful and in that way it would easier to read the erb markup and understand what is being presented on the page. I would suggest renaming these elements to:

- `list_2` => `items_to_be_provided`
- `text_1` => `does_your_client`
- `text_2` => `client_to_provide`

Also you could add a meaningful grouping label to the first list:
```yml
     show:
        about_the_client:
          list: |
            owns a home or has a mortgage (and their values)
            has any savings, investments or other capital (and their values)
            has any other ways to contribute towards legal costs
```

Reply:
the view is definitely much clearer with your suggestions